migrations/sql/V2021.05.26.16.47__update_drilling_and_blasting_activity.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE exploration_surface_drilling ADD COLUMN drill_program character varying(100);
```

Comment:
I think as a team we agreed to not impose length restrictions on data going forward? Should we remove these and the data type is just `varchar`?

Reply:
Yes, I can do that. I was following the restrictions set by vFCBC, but happy to remove.

Also, you are quick!!

Reply:
Updated all to varchar;